cmd/juju/status/output_tabular.go

```diff
@@ -51,8 +51,11 @@ func FormatTabular(writer io.Writer, forceColor bool, value interface{}) error {
 		cloudRegion += "/" + fs.Model.CloudRegion
 	}
 
+	// Default table output
 	header := []interface{}{"Model", "Controller", "Cloud/Region", "Version"}
 	values := []interface{}{fs.Model.Name, fs.Model.Controller, cloudRegion, fs.Model.Version}
+
+	// Optional table output if values exist
 	message := getModelMessage(fs.Model)
 	if message != "" {
 		header = append(header, "Notes")
```

Comment:
hmm, how do we set a note and see what that does? Should notes be the last field in the list of the header?

Reply:
looks like there's messages for upgrade/migration information in the status field so it's not normally there. Can we move the timestamp inboard of that? 

https://github.com/juju/juju/blob/48d0e11e3d90f13a033196f547eb2c0d99c9c3bd/cmd/juju/common/model.go#L51

Reply:
Sure, not a problem at all.